github-profiles/haboufaker-8509/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+    <html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Hi there &#x1f44b;&comma; my name is Hisam Abou Faker</title>
+        <link rel='stylesheet' href='styles.css'/>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
+    </head>
+    <body class="vscode-body vscode-light">
+        <h1 id="hi-there--my-name-is-hisam-abou-faker">Hi there 👋, my name is Hisam Abou Faker</h1>
+<h2 id="i-am-a-developer-and-mechanical-engineer-from-venezuela-living-in-argentina">I am a <strong>Developer</strong> and <strong>Mechanical Engineer</strong> from <strong>Venezuela</strong> living in <strong>Argentina</strong>.</h2>
+<p>I have experience with: HTML, CSS, Javascript,  manual application testing, bug reporting, test execution, ticketing, and experience in web and mobile functional testing in agile environments.</p>
+<p>I am interested in joining multidisciplinary teams focused on innovation and continuous improvement.</p>
+<h2 id="languages-and-tools">Languages and Tools</h2>
+<ul>
+<li>HTML</li>
+<li>CSS</li>
+<li>Bootstrap</li>
+<li>SASS</li>
+<li>JavaScript</li>
+<li>Node.js</li>
+<li>Terminal</li>
+<li>Git</li>
+<li>GitHub</li>
+<li>React</li>
+<li>QA</li>
+<li>Jira</li>
+</ul>
+<h3 id="-you-can-find-me-on">📫 You can find me on:</h3>
```

Comment:
Los id's deben comenzar con una letra. Coloco documentación del W3C al respecto [aquí](https://www.w3.org/TR/html4/types.html)

<img width="1415" alt="imagen" src="https://user-images.githubusercontent.com/41031879/226494830-56621c58-18b5-4cb4-9f36-d07cd38928ca.png">


```suggestion
<h3 id="-you-can-find-me-on">📫 You can find me on:</h3>
```